faq/upgrade-faq.md

```diff
@@ -1,26 +1,42 @@
 ---
-title: FAQs After Upgrade
-summary: Learn about the FAQs after upgrading TiDB.
+title: FAQs Upgrade and After Upgrade
+summary: Learn about some FAQs and their solutions during and after TiDB upgrade.
 aliases: ['/docs/dev/faq/upgrade-faq/','/docs/dev/faq/upgrade/']
 ---
 
-# FAQs After Upgrade
+# FAQs Upgrade and After Upgrade
 
-This document lists some FAQs and their solutions after you upgrade TiDB.
+This document introduces some FAQs and their solutions when or after you upgrade TiDB.
 
-## The character set (charset) errors when executing DDL operations
+## FAQs Upgrade
+
+This section lists some FAQs and their solutions when you upgrade TiDB.
+
+### How are the rolling updates done?
+
+When you apply rolling updates to the TiDB services, the running application is not affected. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3).  If the Pump/Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you update TiDB, Pump is also updated.
+
+### How to upgrade Binary
+
+It is not recommended to deploy TiDB using Binary. The support for upgrading using Binary is not as friendly as using TiUP. It is recommended to [deploy TiDB using TiUP](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup).
```

Comment:
If a file exist in this repository, just put the relative path and do not use the absolute link. Generally, use the corresponding version of link for each branch, so users can stay at that version.
```suggestion
It is not recommended to deploy TiDB using the binary. The TiDB support for upgrading using Binary is not as friendly as using TiUP. It is recommended to [deploy TiDB using TiUP](/production-deployment-using-tiup.md).
```